src/main/java/org/gaul/s3proxy/S3ProxyAccessLogger.java

```diff
@@ -0,0 +1,17 @@
+package org.gaul.s3proxy;
+
+import org.eclipse.jetty.server.Request;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public final class S3ProxyAccessLogger implements AccessLogger {
+    @Override
+    public void logRequest(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) {
+        return;
```

Comment:
This doesn't seem to _do_ anything.  I would expect it to at least include the logging in `S3ProxyHandler.doHandle`:

```
logger.debug("request: {}", request);
logger.debug("header: {}: {}", headerName, Strings.nullToEmpty(headerValue));
```